### PR TITLE
[doc] Document the Paimon action jars in Engine/Flink

### PR DESCRIPTION
**Pull request by klion26**
### Purpose

Add doc about paimon action jars in engine/Flink

### CODE REVIEW

**Comment by klion26** [comment]
cc @JingsongLi

File: docs/content/engines/flink.md [hunk 1]
```diff
@@ -34,27 +34,29 @@ Paimon currently supports Flink 1.17, 1.16, 1.15 and 1.14. We recommend the late
 
 Download the jar file with corresponding version.
 
+> Currently, paimon provides two types jar: one of which(the bundled jar) is used for read/write data, and the other(action jar) for operations such as manually compaction,
+currently action jar supports `compaction`, `drop-partition`, `delete`, `merge-into`, `rollback-to`, `create-tag`, `delete-tag`, `mysql-sync-table`, `mysql-sync-database`, `kafka-sync-table`, `kafka-sync-database`.
```
Comment (JingsongLi) [inline docs/content/engines/flink.md:38]: Don't list this, it is difficult to maintain.

**Review by JingsongLi — APPROVED** [review]
+1

**Comment by klion26** [comment]
@JingsongLi thanks for the review and merging!

**Pull request merged**